src/Error.ts

```diff
@@ -1,4 +1,18 @@
-import {TelnyxRawError} from './Types.js';
+export type TelnyxRawError = {
+  message?: string;
+  type?: string;
+  headers?: {[header: string]: string | string[] | undefined};
+  statusCode?: number;
+  requestId?: string;
+  responseBody?: unknown;
+  code?: string;
+  detail?: string | Error | any;
+  errors?: Array<{
+    code: string;
+    detail: string;
+    title: string;
+  }>;
+};
```

Comment:
this is so these src files are auto sufficient in types for type definitions files to generate correctly without imports or any extra config